sdk/communication/Azure.Communication.Chat/tests/SessionRecords/ChatClientsTest/ReadReceiptGS.json

```diff
@@ -0,0 +1,443 @@
+        "Content-Length": "19",
+        "Content-Type": "application/json",
+        "Date": "Tue, 06 Oct 2020 00:40:48 GMT",
+        "User-Agent": [
+          "azsdk-net-Communication.Administration/1.0.0-alpha.20201005.1",
+          "(.NET Core 4.6.29220.03; Microsoft Windows 10.0.18363 )"
+        ],
+        "x-ms-client-request-id": "bc853c1ff1b685dc47f42059fc987a7a",
+        "x-ms-content-sha256": "J\u002BdoRQjtFVYLx3qOvzptwBLjQWqy6OEWEEk1TY1\u002BrT4=",
+        "x-ms-return-client-request-id": "true"
+      },
+      "RequestBody": {
+        "scopes": [
+          "chat"
+        ]
+      },
+      "StatusCode": 200,
+      "ResponseHeaders": {
+        "api-supported-versions": "2020-07-20-preview1, 2020-07-20-preview2",
+        "Content-Type": "application/json; charset=utf-8",
+        "Date": "Tue, 06 Oct 2020 00:40:48 GMT",
+        "MS-CV": "uKdJpVNQK0\u002B2agYQsmBgrQ.0",
+        "Strict-Transport-Security": "max-age=2592000",
+        "Transfer-Encoding": "chunked",
+        "X-Azure-Ref": "0EL17XwAAAACA1XiTnAMYRayfMy2md3bQWVRPMDFFREdFMDYxMQA5ZmM3YjUxOS1hOGNjLTRmODktOTM1ZS1jOTE0OGFlMDllODE=",
+        "x-ms-client-request-id": "bc853c1ff1b685dc47f42059fc987a7a",
+        "X-Processing-Time": "283ms"
+      },
+      "ResponseBody": {
+        "id": "8:acs:fa5c4fc3-a269-43e2-9eb6-0ca17b388993_00000005-9b3c-2cec-e7a1-1c4822000fb7",
+        "token": "Sanitized",
+        "expiresOn": "2020-10-06T17:40:48.068365-07:00"
+      }
+    },
+    {
+      "RequestUri": "https://chat-prod-e2e.communication.azure.com/chat/threads?api-version=2020-09-21-preview2",
+      "RequestMethod": "POST",
+      "RequestHeaders": {
+        "Accept": "application/json",
+        "Authorization": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c",
+        "Content-Length": "234",
+        "Content-Type": "application/json",
+        "User-Agent": [
+          "azsdk-net-Communication.Chat/1.0.0-alpha.20201005.1",
+          "(.NET Core 4.6.29220.03; Microsoft Windows 10.0.18363 )"
+        ],
+        "x-ms-client-request-id": "580688fefb8bbcd1b08931959c4a5643",
+        "x-ms-return-client-request-id": "true"
+      },
+      "RequestBody": {
+        "topic": "Thread topic - ReadReceipts Test",
+        "members": [
+          {
+            "id": "8:acs:fa5c4fc3-a269-43e2-9eb6-0ca17b388993_00000005-9b3c-2a12-e7a1-1c4822000fb6"
+          },
+          {
+            "id": "8:acs:fa5c4fc3-a269-43e2-9eb6-0ca17b388993_00000005-9b3c-2cec-e7a1-1c4822000fb7"
+          }
+        ]
+      },
+      "StatusCode": 207,
+      "ResponseHeaders": {
+        "api-supported-versions": "2020-09-21-preview2",
+        "Content-Type": "application/json; charset=utf-8",
+        "Date": "Tue, 06 Oct 2020 00:40:49 GMT",
+        "Location": "https://52.168.174.23/chat/threads/19%3Ab60636747c204956973c8c21b22ab97e@thread.v2",
```

Comment:
sanitize

Reply:
same ^^

Reply:
Actually, just posted that question because this is an internal IP. Will sanitize if the azure folks recommend it. Thanks for pointing it out.

Reply:
ok, that was quick:
https://teams.microsoft.com/l/message/19:7b87fb348f224b37b6206fa9d89a105b@thread.skype/1602023801069?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&parentMessageId=1601940246913&teamName=Azure%20SDK&channelName=Language%20-%20DotNet&createdTime=1602023801069